packages/core-api/src/routing/FlatRoutes.tsx

```diff
@@ -71,6 +72,15 @@ type FlatRoutesProps = {
 };
 
 export const FlatRoutes = (props: FlatRoutesProps): JSX.Element | null => {
+  const app = useApp();
+  const { NotFoundErrorPage } = app.getComponents();
   const routes = createRoutesFromChildren(props.children);
+
+  // TODO(Rugvip): Possibly add a way to skip this, like a noNotFoundPage prop
```

Comment:
What would be the benefit of just skipping it?

Reply:
To be able to render no content or a different not found page, although tbh that's prolly possible to do with a catch-all route in the children